### PR TITLE
Create UserController

### PR DESCRIPTION
**Pull request by Xypus**
#### Description

Created a UserController and UserView with '.me' action that returns current user's data. Plus specs
- [x] ready for review
#### Approved:
- [ ] @mikolajgrzyb 
- [ ] @pjar 
- [ ] @marcinwyszynski

### CODE REVIEW

File: spec/controllers/user_controller_spec.exs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+defmodule Janitor.UserControllerSpec do
+  use ESpec.Phoenix, controller: Janitor.UserController
+  import Janitor.UserFactory
+
+  describe "me" do
+    let :user, do: create(:user)
+    let :custom_conn, do: assign(conn, :current_user, user)
```
Comment (mikolajgrzyb) [inline spec/controllers/user_controller_spec.exs:7]: @Xypus Maybe we could create some helper out of it?

Comment (Xypus) [inline spec/controllers/user_controller_spec.exs:7]: @mikolajgrzyb Makes sense.

File: Dockerfile [hunk 2]
```diff
@@ -8,3 +8,5 @@ WORKDIR $HOME
 
 COPY mix.exs mix.exs
 RUN yes | mix deps.get
+RUN apt-get update
+RUN yes | apt-get install erlang-tools
```
Comment (marcinwyszynski) [inline Dockerfile]: `apt-get -y install erlang-tools`

File: web/views/user_view.ex [hunk 3]
```diff
@@ -0,0 +1,9 @@
+defmodule Janitor.UserView do
+  use Janitor.Web, :view
+
+  def render("me.json", %{user: user}) do
```
Comment (marcinwyszynski) [inline web/views/user_view.ex:4]: What's the point of having a parameter here which will never change? Can't you just pass `user` here and be done with that?

File: web/plugs/user_authorizer.ex [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    )
+  end
+
+  #PRIVATE
+
+  defp perform_checks({conn, nil}, []), do: end_processing(conn, nil)
+  defp perform_checks({conn, nil}, [check_fun | remaining_checks]) do
+    perform_checks(check_fun.(conn), remaining_checks)
+  end
+  defp perform_checks({conn, err}, _), do: end_processing(conn, err)
```
Comment (marcinwyszynski) [inline web/plugs/user_authorizer.ex]: 🍪

File: web/plugs/user_authorizer.ex [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+  defp end_processing(conn, err) do
```
Comment (marcinwyszynski) [inline web/plugs/user_authorizer.ex]: To be fair this function is useless. These cases could be handled directly in bodies 1 and 3 of the `perform_checks` function:

``` elixir
defp perform_checks({conn, nil}, []) do: conn
# ...
defp perform_checks({conn, err}, _), do
  conn |> send_resp(403, "Unauthorized request") |> Plug.Conn.halt
end
```

File: web/views/user_view.ex (hunk 3 above)
Comment (mikolajgrzyb) [inline web/views/user_view.ex:4]: yeah, "me.json" doesn't need user. We got it from JWT.

**Pull request merged**